src/NumericalAlgorithms/Interpolation/PredictedZeroCrossing.hpp

```diff
@@ -0,0 +1,19 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <vector>
+
+namespace intrp {
+
+/*!
+ * \brief Predicts the zero crossing of a function.
+ *
+ * Fits a linear function to a set of values at different times
+ * and uses the fit to predict what time the value zero will be crossed.
```

Comment:
don't reference time in the documentation. That is replace:
- time(s) -> x_value(s)
- value(s) -> y_value(s)